CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Server and open synchronized Realms (#1276).
 
 ### Bug fixes
 * Removed a false negative warning when using `User.createConfiguration`.
+* Fix the type definition for `User.authenticate`.
```

Comment:
"Fix" -> "Fixed"

Is there an issue to refer to?

Reply:
Not really, I just ran into it while trying to use the API in studio